genai_bench/distributed/runner.py

```
@@ -276,7 +277,14 @@ def _create_metrics_handler(self) -> MessageHandler:
 
         def handler(environment: Environment, msg: Any, **kwargs) -> None:
             # Master receives and aggregates metrics
-            metrics = RequestLevelMetrics.model_validate_json(msg.data)
+            try:
+                metrics = RequestLevelMetrics.model_validate_json(msg.data)
+            except ValidationError as e:
+                logger.warning(
```

Why do we drop the metrics instead of failing fast?